kerberos_helper.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import re
+
+from clusterdock.utils import wait_for_condition
+
+DEFAULT_OPERATING_SYSTEM = '6.6'
+IMAGE_NAME_TEMPLATE = 'clusterdock/topology_nodebase_kerberos:centos{}'
+
+KERBEROS_CONFIG_CONTAINER_DIR = '/etc/clusterdock/kerberos'
+
+KDC_ACL_FILENAME = '/var/kerberos/krb5kdc/kadm5.acl'
+KDC_CONF_FILEPATH = '/var/kerberos/krb5kdc/kdc.conf'
+KDC_HOSTNAME = 'kdc'
+KDC_GROUPNAME = 'kdc'
+# Following is the location clusterdock places the keytab file of kerberos_principals
+# so that client nodes like SDC docker can fetch it.
+KDC_KEYTAB_FILEPATH = '{}/clusterdock.keytab'.format(KERBEROS_CONFIG_CONTAINER_DIR)
+KRB5_CONF_FILEPATH = '/etc/krb5.conf'
+MAPR_CONF_PATH = '/opt/mapr/conf'
+MAPR_KEYTAB_FILENAME = 'mapr.keytab'
+# Following is the location clusterdock places the mapr.keytab file so that client nodes like SDC docker can fetch it.
+MAPR_KEYTAB_FILEPATH = '{}/{}'.format(KERBEROS_CONFIG_CONTAINER_DIR, MAPR_KEYTAB_FILENAME)
+# Following is the location expected by MapR for the mapr.keytab file.
+MAPR_CONF_KEYTAB_FILEPATH = '{}/{}'.format(MAPR_CONF_PATH, MAPR_KEYTAB_FILENAME)
+MAPR_PRINCIPAL = 'mapr/my.cluster.com'
+LINUX_USER_ID_START = 1000
+
+logger = logging.getLogger('clusterdock.{}'.format(__name__))
+
+
+class Kerberos_Helper:
+
+    def __init__(self, network, operating_system):
```

Comment:
Can we add a docstring to the class reflecting this __init__? Also, since it looks like `operating_system` is a version and not the whole OS name, can we update the naming to make that clear?

Reply:
Done.